drivers/dma/Kconfig.mcux

```diff
@@ -0,0 +1,90 @@
+# Kconfig - MCUXpresso SDK DMA
+#
+#
+#
+# Copyright (c) 2017, Phytec Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig DMA_MCUX
```

Comment:
Reorder your commits so the Kconfig gets introduced before the shim, or just squash the Kconfig into the shim commit